### PR TITLE
Thunderbird for Android 8.0b4

### PR DESCRIPTION
**Pull request by kewisch**

### CODE REVIEW

File: android_beta/8.0beta.yml [hunk 1]
```diff
@@ -1,12 +1,13 @@
 release:
   release_date: '2024-09-30'
   text: |
-    **These notes apply to Thunderbird for Android version 8.0 beta 3 released October, 14, 2024.**
+    **These notes apply to Thunderbird for Android version 8.0 betas**
```
Comment (coreycb) [inline android_beta/8.0beta.yml:4]: This is probably fine but we use this line to mention the release date of the latest beta.

**Review by coreycb — APPROVED** [review]

**Comment by coreycb** [comment]
Looks fine to me. We'll need to revisit these notes if, someday we end up publishing them to the website, to ensure they render correctly.

File: android_beta/8.0beta.yml (hunk 1 above)
Comment (kewisch) [reply android_beta/8.0beta.yml:4]: Ah good to know, I was wondering if it was jut out of date or part of testing. Can update this in the next beta

**Comment by Sancus** [comment]
Yeah, some code changes will be required for this. The Android notes will need their own namespace.

**Pull request merged**